src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail_controller.js

```diff
@@ -108,6 +106,18 @@ export default class ReplicationControllerDetailController {
      * @export {boolean}
      */
     this.eventsOrder = DOWNWARDS;
+
+    /**
+     * If true then sidebar will be displayed, otherwise it won't.
+     * @export {boolean}
+     */
+    this.displaySidebar = this.mdMedia('gt-sm');
+
+    /**
+     * Binding resize listener, which switches displaySidebar boolean variable.
+     */
+    angular.element(this.window)
```

Comment:
Yes like here https://github.com/kubernetes/dashboard/blob/master/src/app/frontend/deploy/filereader_directive.js#L32
But it requires also to encapsulate md-tab in your directive to make it work.